lib/active_merchant/billing/gateways/omise.rb

```diff
@@ -194,8 +194,10 @@ def post_data(parameters)
 
       def https_request(method, endpoint, parameters=nil, options={})
         raw_response = response = nil
+        req_headers  = headers(options)
```

Comment:
I think that's a good option too, if so, I may have to write something like this.

```
def headers(options={})
   key = options[:key] || @secret_key
   h = {
     'Content-Type'    => 'application/json;utf-8',
     'User-Agent'      => "ActiveMerchantBindings/#{ActiveMerchant::VERSION} Ruby/#{RUBY_VERSION}",
     'Authorization'   => 'Basic ' + Base64.encode64(key.to_s + ':').strip,
     'Accept-Encoding' => 'utf-8'
   }
   h['Omise-Version'] = @api_version if @api_version
   h
 end
```
Does this look ok?